third_party/musl/src/string/stpcpy.c

```diff
@@ -7,6 +7,9 @@
 #define HIGHS (ONES * (UCHAR_MAX/2+1))
 #define HASZERO(x) ((x)-ONES & ~(x) & HIGHS)
 
+#ifdef __GNUC__
+__attribute__((no_sanitize("address")))
+#endif
```

Comment:
Bit curious, is there a TODO to fix this later , or this method wont be processed by the adresss sanitizer anytime in the future, if so , why is that ?